geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -45,12 +46,13 @@ object HBaseFeatureIndex extends HBaseIndexManagerType {
   val DataColumnQualifierDescriptor = new HColumnDescriptor(DataColumnQualifier)
 
   case class ScanConfig(hbaseFilters: Seq[HBaseFilter],
-                        entriesToFeatures: Iterator[Result] => Iterator[SimpleFeature])
+                        entriesToFeatures: Iterator[Result] => Iterator[SimpleFeature],
+                        ecql: Option[Filter])
```

Comment:
i don't think we need the ecql here. it will either be in the hbaseFitlers for server side config, or wrapped in the entriesToFeatures for client side.